js/RichText/UserMention.js

```diff
@@ -47,11 +47,13 @@ window.GLPI.RichText.UserMention = class {
     * @param {Editor} editor
     * @param {number} activeEntity
     * @param {string} idorToken
+    * @param {Array} mentions_options
```

Comment:
To use the same naming convention as other params from the same method.

```suggestion
    * @param {Array} mentionsOptions
```